service-a.py

```
@@ -6,6 +6,18 @@
 from typing import Optional
 import secrets
 import os
+from config import (
+    FRONTEND_EXTERNAL_IP,
+    FRONTEND_PORT,
+    EXTERNAL_IP,
+    CONSENT_STORE_INTERNAL_URL,
```

Ruff (0.11.9)

`config.CONSENT_STORE_INTERNAL_URL` imported but unused

Remove unused import: `config.CONSENT_STORE_INTERNAL_URL`

(F401)